### PR TITLE
changelog: add v0.1.0

### PR DESCRIPTION
**Pull request by Wondertan**
Adds changelog of features and quick description of the first-ever release of celestia-node.

### CODE REVIEW

**Comment by Wondertan** [comment]
@renaynay, pls add more description to release if you feel so.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,21 +1,58 @@
...
-- [go package] (Link to PR) Description @username
-
-### IMPROVEMENTS
-
-- [go package] (Link to PR) Description @username
-
-### BUG FIXES
-
-- [go package] (Link to PR) Description @username
+- [das: das: Log out square width after sampling #254](https://github.com/celestiaorg/celestia-node/pull/254) [@Wondertan](https://github.com/Wondertan)
```
Comment (liamsi) [inline CHANGELOG.md:13]:
```suggestion
- [das: Log out square width after sampling #254](https://github.com/celestiaorg/celestia-node/pull/254) [@Wondertan](https://github.com/Wondertan)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,21 +1,58 @@
-# Unreleased Changes
+# CHANGELOG
 
-## vX.Y.Z
+## v0.1.0 | 2021-11-3
+This is the first `celestia-node` release. The release gives a birth to "halo" data availability p2p network with 
+respective features and lays down a foundation for future Celestia network.
```
[the remark this answers is not in the document]
Comment (Wondertan) [reply CHANGELOG.md:5]: I like your suggestion. One thing is that I still would like to keep a foundation word as it truly is.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,21 +1,58 @@
...
 
-Month, DD, YYYY
-
-### BREAKING CHANGES
-
-- [go package] (Link to PR) Description @username
+Mainly, we introduce 3 packages:
+* `service/header` - defines everything related to headers Data Structures, synchronization logic and p2p exchange logic
+* `service/share` - defines everything related to Shares(pieces of shared Block data), the p2p networking logic and sampling
+* `node` - keeps central Node singleton with its assembly, essentially glueing all the bits together.
```
[the remark this answers is not in the document]
Comment (Wondertan) [reply CHANGELOG.md:10]: Telling Header is sufficient, ExtendedHeader is very context specific thing that most readers won't have cluer about.

[the remark this answers is not in the document]
Comment (Wondertan) [reply CHANGELOG.md:10]: "requesting and sampling" ... "___to___/from network"?

[the remark this answers is not in the document]
Comment (renaynay) [reply CHANGELOG.md:10]: changed to/from

**Review by renaynay — APPROVED** [review]

**Review by adlerjohn — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,21 +1,59 @@
-# Unreleased Changes
+# CHANGELOG
 
-## vX.Y.Z
+## v0.1.0 | 2021-11-3
```
Comment (adlerjohn) [inline CHANGELOG.md:3]: Small nit
```suggestion
## v0.1.0 | 2021-11-03
```

**Pull request merged**